### DISCUSSION TITLE
Option to disable Pieces OS startup pop-up

### DISCUSSION THREAD

**Discussion by RakeshPotnuru** [pieces-app/support#125] [Ideas]
I think it would be nice to give an option to disable this pop-up on startup.

![image](https://github.com/pieces-app/support/assets/70439799/dbd9d764-84c4-45f8-9961-5659cbedfa68)

**Comment by caleb-at-pieces** [comment]
Seems to be a duplicate of https://github.com/pieces-app/support/discussions/114

**Comment by RakeshPotnuru** [comment]
Yes, sorry. I actually submitted a typeform for this and I was told to open a discussion but I didn't find that in search before opening discussion.
